react-native-libraries.json

```diff
@@ -13851,5 +13851,18 @@
     "android": true,
     "windows": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/kuatsu/react-native-boost/tree/master/packages/react-native-boost",
+    "npmPkg": "react-native-boost",
```

Comment:
```suggestion
```

It can be skipped:
* https://github.com/react-native-community/directory?tab=readme-ov-file#npmpkg